core/domain/exp_services.py

```diff
@@ -907,9 +902,9 @@ def _create_exploration(
         committer_id: str. The id of the user who made the commit.
         exploration: Exploration. The exploration domain object.
         commit_message: str. The commit description message.
-        commit_cmds: list(dict). A list of commands, describing changes
-            made in this model, which should give sufficient information to
-            reconstruct the commit. Each dict always contains:
+        commit_cmds: list(ExplorationChange). A list of commands, describing
+            changes made in this model, which should give sufficient information
+            to reconstruct the commit. Each object always contains:
```

Comment:
If you are passing in a list of domain objects then the fact that the domain object is of the given type is sufficient to understand its structure. You don't need to enumerate its internal structure further (this is a strength of using this pattern).

Reply:
Done.